src/ansi.rs

```diff
@@ -668,6 +668,16 @@ pub trait Handler {
 
     /// Set hyperlink.
     fn set_hyperlink(&mut self, _: Option<Hyperlink>) {}
+
+    /// Set mouse cursor.
+    ///
+    /// The requested cursor shapes usually have a name from CSS specificiation
+    /// <https://www.w3.org/TR/css-ui-3/#cursor>.
+    ///
+    /// Some legacy clients could send a legacy X11 names, but it's encouradges
+    /// to use CSS values and fallback to default cursor or ignore requets when
+    /// the name is not the one from the said specification.
```

Comment:
```suggestion
    /// Some legacy clients could send a legacy X11 name, but it's encouraged
    /// to use CSS values and fall back to the default cursor or ignore requests when
    /// the name is not one from the said specification.
```

I'd also say that "fall back to default or ignore requests" is not precise enough. There should be **one** correct behavior here, not two. This must be clearly defined.